apertium-tur.tur.lexc

```diff
@@ -24198,9 +24200,10 @@ azmet:azmet V-AR-IV ; ! "determine" (DIR)
 bahset:bahset V-AR-TV ; ! "talk about" (DIR)
 bahşet:bahşet V-AR-TV ; ! "forgive" (DIR)
 bak:bak V-AR-IV ; ! "look at" (DIR)
-ban:ban V-AR-TV ; ! "dip into" (DIR)
+bakın:bakın V-AR-IV ; ! "look around"
+ban:ban V-AR-IV ; ! "dip into" (DIR)
 bas:bas V-AR-TV ; ! "press" (DIR)
-bas:bas V-AR-TV ; ! "raid" (DIR)
+bas:bas V-AR-IV ; ! "raid" (DIR)
```

Comment:
this TV
evi su bastı -> TV

Reply:
All right, I put it back as TV.
But I changed it because you told it IV above in your comment:
> "bas" ="bastır"="baskın", here it is intransitive

Reply:
okay was my fault "bas" !="bastır"
bas=print ->TV
bastır=suppress ->TV